src/rest/resources.ts

```diff
@@ -294,61 +307,59 @@ export class Resources implements IResources {
             method: 'GET',
             baseURL: this.baseUrl,
             url: `/oauth/token?oauth_provider=${oAuthProvider}`,
-            headers: this.getHeadersWithAuthorization(this.userToken)
+            headers: this.headers,
         });
     }
 
     public getOAuthProviders(): AxiosPromise<any[]> {
         return this.axios.request<any[]>({
             method: 'GET',
-            headers: this.getHeadersWithAuthorization(this.userToken),
+            headers: this.headers,
             baseURL: this.baseUrl,
-            url: '/oauth'
+            url: '/oauth',
         });
     }
 
-    public updateActivity(workspaceId: string): AxiosPromise<void> {
+    public updateActivity(workspaceId: string): AxiosPromise<void> { // -
```

Comment:
What is that comment at the end of line for? Shouldn't it be removed?

Reply:
It should be removed